controllers/healthcheck.go

```diff
@@ -51,7 +47,11 @@ func (r *EtcdadmClusterReconciler) performEndpointHealthCheck(ctx context.Contex
 
 	client := r.etcdHealthCheckConfig.etcdHttpClient
 	healthCheckURL := getMemberHealthCheckEndpoint(endpoint)
-	r.Log.Info(fmt.Sprintf("Performing healthcheck on endpoint %s", healthCheckURL))
+	if !periodicCheck {
+		// logging non-failures only for non-periodic checks so as to not log too many events
```

Comment:
with the periodic healthcheck loop the logs were getting filled with these messages. For periodic healthcheck we should only log failures to avoid too many logs